components/compositing/tracing.rs

```diff
@@ -57,6 +57,7 @@ mod from_constellation {
                 Self::GetClientWindowRect(..) => target!("GetClientWindowRect"),
                 Self::GetScreenSize(..) => target!("GetScreenSize"),
                 Self::GetAvailableScreenSize(..) => target!("GetAvailableScreenSize"),
+                Self::CollectReport(..) => target!("CollectReport"),
```

Comment:
Do you mind including `Memory` in the message name here?

```suggestion
                Self::CollectMemoryReport(..) => target!("CollectMemoryReport"),
```